src/utils.jl

```diff
@@ -92,7 +91,7 @@ end
     push!(q.args, :(i, $tup))
     q
 end
-@generated store!(p::Ptr{UInt}, tup::T, i) where {T<:Tuple} = isbitstype(T) ? :(_store!(p, tup, i)) : :(store!(p+i, tup); i + sizeof(tup))
+@generated store!(p::Ptr{UInt}, tup::T, i) where {T<:Tuple} = :(_store!(p, tup, i))
```

Comment:
Mind replacing this with:
```julia
@inline store!(p::Ptr{UInt}, tup::T, i) where {T<:Tuple} = _store!(p, tup, i)
```
Or perhaps deleting this and renaming `_store!` -> `store!`?

Reply:
Yeah renaming is nice and clean